ml-agents/mlagents/trainers/components/bc/model.py

```diff
@@ -1,4 +1,5 @@
-import tensorflow as tf
+from mlagents.trainers import tf
```

Comment:
@ervteng @awjuliani @andrewcoh (research pod folks) How do you feel about importing this way instead (instead of `import tensorflow as tf`)? Would something like `import mlagents.tensorflow as tf` be nicer?

Note that either way, we should prevent directly importing "raw" tensorflow except for one central spot; I added some checks to flake8 to prevent this.

Reply:
Either should be OK. I kind of like the current way rather than the other one since it seems less like we have a "special" version of TensorFlow built into mlagents.

Reply:
By "current way", do you mean the one in the PR now, or on develop?

I feel like `mlagents.trainers` is the wrong place for it; maybe `mlagents.utils` or something like that?

Reply:
The one in the PR. 

There could definitely be an `mlagents.tf_utils` file - there are actually some other generic trainer functions that could go in there (e.g. `get_variable` in the SAC model, any checkpointing functionality, etc.).

Reply:
Moved the import wrapper to `mlagents.tf_utils`